src/Database/Expression/Comparison.php

```diff
@@ -75,7 +75,7 @@ class Comparison implements ExpressionInterface, FieldInterface
      * @param string|array|null $type the type name used to cast the value
      * @param string $operator the operator used for comparing field and value
      */
-    public function __construct($field, $value, $type, string $operator)
+    public function __construct($field, $value, $type = null, string $operator = '=')
```

Comment:
I wonder if this was more readable if we converted the magic string `=` operator char into a speaking class const?
```php
public const EQUALS = '=';
```
and 
```php
__construct($field, $value, $type = null, string $operator = self::EQUALS)
```

Especially if this char is used a bit more also as input from other sides.
But could also be a bit overkill.

Reply:
This class is too lower level to bother with constants. Also it's not practical to add constants for all possible operators so adding it just for `=` would just create inconsistency.